meta/system.py

```diff
@@ -303,6 +303,26 @@ def plp_features(self, corpus, num_deriv=2, num_features=23, **kwargs):
             )
         self.feature_flows[corpus]["uncached_plp"] = f.feature_flow
 
+    def tone_features(self, corpus, prefix="", **kwargs):
+        timestamp_flow = self.feature_flows[corpus][
+            kwargs.pop("timestamp_flow", "energy")
```

Comment:
Why is timestamp flow "hidden" in kwargs? Wouldn't it be better to have that as an explicit arg for `tone_features`?